aaf_monitor_logs/cmake/FindMongoClient.cmake

```diff
@@ -0,0 +1,78 @@
+# - Find MongoClient; NOTE: this is specific to warehouse_ros!
```

Comment:
again, this file is not needed, is it? There is no CPP code in here... Also `CMakeLists.txt` should be fixed to only have the right deps in it.